web/src/beta/lib/core/engines/Cesium/index.tsx

```diff
@@ -185,6 +185,8 @@ const Cesium: React.ForwardRefRenderFunction<EngineRef, EngineProps> = (
         useWebVR={!!property?.default?.vr || undefined}
         light={light}
         useDepthPicking={false}
+        maximumRenderTimeChange={0}
+        requestRenderMode={true}
```

Comment:
Why should we set these property to scene as well?

Reply:
Removed. I think it's unnecessary since we already set it on Viewer.